tests/python/test_disallowed_identifiers.py

```diff
@@ -73,12 +73,12 @@ def test_check_identifiers_attribute(mock_ignore_check):
 
 @patch('python_hooks.disallowed_identifiers.print')
 def test_check_identifiers_attribute_and_ignore(mock_print):
-    actual_return = check_identifiers(FILE, Identifier.attribute, ['split'], ['splitlines'])
+    actual_return = check_identifiers(FILE, Identifier.attribute, ['disallowed'], ['allowed'])
     assert actual_return == 1
-    mock_print.call_args_list == [
+    assert mock_print.call_args_list == [
```

Comment:
ah yeah, was wondering how the test passed without the correct params... 

I guess we need a linter check for no assert in tests!

Reply:
lol